cfgov/v1/models/filterable_page.py

```diff
@@ -13,6 +13,19 @@
 from v1.util.ref import get_category_children
 
 
+class SearchResultsPaginator(Paginator):
+    def _get_page(self, object_list, *args, **kwargs):
+        if object_list and object_list.to_queryset:
+            object_list = (
+                object_list.to_queryset()
+                .specific()
+                .live()
```

Comment:
@willbarton calling this out specifically as a last-minute way to filter out any not-live pages that may still be in the ES index for some reason.

Although the current code [does try to filter out not-live pages as part of the ES query](https://github.com/cfpb/consumerfinance.gov/blob/01e024e65ad2fd5f73300edd0732099e2576e05e/cfgov/v1/documents.py#L110), it doesn't actually check the status [at the place those results get converted to Page objects](https://github.com/cfpb/consumerfinance.gov/blob/01e024e65ad2fd5f73300edd0732099e2576e05e/cfgov/v1/documents.py#L200).